docs/scos/user/technology-partners/202108.0/payment-partners/bs-payone/bs-payone.md

```diff
@@ -134,27 +133,72 @@ $container[static::PAYMENT_METHOD_HANDLER] = function () {
  };
 ```
 
-## Integration with Payment module (PaymentDependencyProvider):
+The project (demoshop) level `\Pyz\Zed\Checkout\CheckoutDependencyProvider` method `provideBusinessLayerDependencies` container has to be extended with the checkout plugins of Payone. It can be done by adding plugins into related plugins stacks in these methods in `\Pyz\Zed\Checkout\CheckoutDependencyProvider`:
```

Comment:
```suggestion
The project (Demo Shop) level `\Pyz\Zed\Checkout\CheckoutDependencyProvider` method `provideBusinessLayerDependencies` container has to be extended with the checkout plugins of Payone. This can be done by adding plugins to the related plugins stacks in these methods in `\Pyz\Zed\Checkout\CheckoutDependencyProvider`:
```